fusion/python/sample.py

```diff
@@ -174,10 +181,13 @@ def query_api(term, location):
     print(u'{0} businesses found, querying business info ' \
         'for the top result "{1}" ...'.format(
             len(businesses), business_id))
-    response = get_business(bearer_token, business_id)
```

Comment:
Could you undo the changes to `query_api` ? This is supposed to be a _very_ basic example program, just printing the top result keeps the example simple and easy to read, adding more complicated logic distracts from the functionality it is trying to highlight